.github/workflows/test-job.yaml

```diff
@@ -106,9 +106,21 @@ jobs:
       uses: docker-practice/actions-setup-docker@master
 
     - name: Build Docker Image
-      run: make docker-image
+      run: IMG=kuberay/operator:nightly make docker-image
```

Comment:
@wilsonwang371 one comment from my side.

I suggest to use commit id as the tag and it's easier to target the problem. 

1. for PR changes, let's use local image and load into kind for testing but skip pushing to registry
2. for merge change, we can push to registry and retag to nightly and push it again.